Src/TestTypeFoundation/TypeWithCastOperatorsWithoutPublicConstructor.cs

```diff
@@ -6,12 +6,12 @@ private TypeWithCastOperatorsWithoutPublicConstructor()
         {
         }
 
-        public static implicit operator TypeWithCastOperatorsWithoutPublicConstructor(int _)
+        public static implicit operator TypeWithCastOperatorsWithoutPublicConstructor(int ignored)
         {
             return new TypeWithCastOperatorsWithoutPublicConstructor();
         }
 
-        public static explicit operator TypeWithCastOperatorsWithoutPublicConstructor(string _)
+        public static explicit operator TypeWithCastOperatorsWithoutPublicConstructor(string ignored)
```

Comment:
I think `ignored` is more clear than `_` so it's no confused with the [wildcard pattern](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/pattern-matching).

Reply:
:+1: